client/app/achievements/achievements.less

```diff
@@ -1,3 +1,7 @@
 ng-invald {
   color: red
 }
+
+.event-description > img {
+  width: 100%;
```

Comment:
If the image is unstyled I don't want them blowing out of the box but if additonal styling is applied it should accept that. What case do you think this would break?